File: 1-Iniciante/1040-Media3-5/1040.cs
using System; 

class URI {

    static void Main(string[] args) { 
    
        string[] valores = Console.ReadLine().Split(' ');
        
        double valor1, valor2, valor3, valor4;

        valor1=double.Parse(valores[0])*2;
        valor2=double.Parse(valores[1])*3;
        valor3=double.Parse(valores[2])*4;
        valor4=double.Parse(valores[3]);
        Console.WriteLine($"Media: {Math.Round((valor1+valor2+valor3+valor4)/10, 1):f1}");
        if((valor1+valor2+valor3+valor4)/10>=7){
            Console.WriteLine("Aluno aprovado.");
        }
        else if((valor1+valor2+valor3+valor4)/10<5){
            Console.WriteLine("Aluno reprovado.");
        }
        else{
            Console.WriteLine("Aluno em exame.");
            
            double nova_nota = double.Parse(Console.ReadLine());
            Console.WriteLine($"Nota do exame: {nova_nota:f1}");
            
            if ((((valor1+valor2+valor3+valor4)/10)+nova_nota)/2>=5){
                Console.WriteLine("Aluno aprovado.");
            }
            else{
                Console.WriteLine("Aluno reprovado.");
            }
            
            
            Console.WriteLine($"Media final: {Math.Round((((valor1+valor2+valor3+valor4)/10)+nova_nota)/2, 1):f1}");
        }
        
        
    
    }

}

File: 1-Iniciante/1040-Media3-5/1040.py
x=str(input())
x=x.split(" ")
n1=float(x[0])
n2=float(x[1])
n3=float(x[2])
n4=float(x[3])
med=(n1*2+n2*3+n3*4+n4)/10
print(f"Media: {med:.1f}")
if med>=7:
    print("Aluno aprovado.")
elif med>=5:
    print("Aluno em exame.")
    x=float(input())
    print(f"Nota do exame: {x:.1f}")
    med=(med+x)/2
    if med>=5:
        print("Aluno aprovado.")
    else:
        print("Aluno reprovado.")
    print(f"Media final: {med:.1f}")
else:
    print("Aluno reprovado.")